apple-pie/styles/style.css

```diff
@@ -0,0 +1,58 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto+Condensed:700|Roboto:100,300,700");
+
+body {
+  font-family: Roboto;
+}
+
+h1 {
+  text-align: center;
+  position: relative;
+  top: 400px;
+  color: white;
+  font-size: 50px;
+  letter-spacing: 0.9px;
+}
+header {
+  width: 960px;
+  height: 900px;
+  background: center / contain url("../images/apple-pie.jpg") no-repeat;
+  margin: auto;
```

Comment:
It seems that "contain" (background-size) is the reason why you have a top margin. To remove it, you could add a negative top margin:
`margin-top: -40px`